query/src/main/scala/filodb/query/PromQueryResponse.scala

```diff
@@ -8,14 +8,20 @@ final case class RemoteErrorResponse(status: String, errorType: String, error: S
 
 final case class ErrorResponse(errorType: String, error: String, status: String = "error") extends PromQueryResponse
 
-final case class SuccessResponse(data: Data, status: String = "success") extends PromQueryResponse
+final case class SuccessResponse(data: Data, status: String = "success",
+                                 isPartial: Option[Boolean] = None,
+                                 message: Option[String] = None) extends PromQueryResponse
```

Comment:
was the plan to use http response codes to indicate partial results, or attributes in json ?

Reply:
Both. Http response code will be 206 and also, json will have a flag which can be used in grafana later if needed.